### PR TITLE
support d3 v4 and v3

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

**Comment by cheapsteak** [comment]
This actually work?

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "react": "*"
   },
   "dependencies": {
-    "d3": "^3.5.17",
+    "d3": "*",
```
Comment (shanewilson) [inline package.json]: should have a version here - this will just default to the latest one

Comment (shanewilson) [inline package.json]: made a bug - https://github.com/knitjs/knit/issues/18

should be able to set

```
"peerDependencies": {
  "d3": ">=3"
}
```

in survival/package.json when that is closed

Comment (Jephuff) [inline package.json]: So should I just leave this for now then?

**Comment by Jephuff** [comment]
@cheapsteak I sure hope it works

**Comment by cheapsteak** [comment]
LTGM

**Pull request merged**